### ISSUE TITLE
Add Berhu Loss & Use with Flux.jl?

### ISSUE THREAD

**Issue by jchen975** [JuliaML/LossFunctions.jl#115]
Hi there, 

I modified your Huber loss to create the Berhu loss according to [this paper](https://statweb.stanford.edu/~owen/reports/hhu.pdf) (generalized formula, B_M(z) on p.4). I was wondering if you would like to add it to this library (I'm not familiar with how github pull request works). 

Also, I'd appreciate some advice on how to use loss functions provided here in projects using Flux.jl. In particular, if I want to use a different crossover parameter for every batch, can I do the following? 
```julia
opt = ADAM()
loss = HuberLoss()
for i = 1:epochs
    for j = 1:numBatches
        loss.d = 0.2*maximum(abs(model(batchX) - batchY))
        Flux.train!(loss, Flux.params(model), [(batchX, batchY)], opt)
     end  # omitted indexing for next batch, epoch ++, etc. 
end
```

**EDIT** 
I removed my implementation of Berhu loss. I made a few mistakes earlier, so I'll post it again once I correct them.

**Comment by juliohm** [comment]
Please direct questions of use to our machine-learning stream on Zulip.
